R/paginate.R

```diff
@@ -56,6 +56,11 @@ req_paginate <- function(req,
 
 #' Perform a paginated request
 #'
+#' After preparing a paginated request with [req_paginate()] perform it with
+#' [paginate_req_perform()]. In case the requests were interrupted, you can
```

Comment:
Would it be better to copy the `multi_req_perform()` interface here? i.e. `paginate_req_perform()` should always succeed but sometimes return an error object? Or maybe in this case it's sufficient to return a shorter list than expected with a warning.

You could use the same technique as in `pool_run()` to handle user interrupts.